### DISCUSSION TITLE
Add member to specific private channel

### DISCUSSION THREAD

**Discussion by PrismPaw** [DSharpPlus/DSharpPlus#1312] [Ideas]
This is partially a question, and idea about a function that maybe I missed (if it doesn't exist)

Looking at discord bots made in other languages it seems node.js has an ability to add users to a specific private channel, (I'm decently new to Dsharp so I could be completely missing this if it exists -- please point me in the right direction if so!) But looking at Dsharp I can't locate this functionality. This could be super useful if it's not already a feature!

**Comment by JulianusIV** [comment]
If i understood what you want to achive correctly, you should look at the DiscordChannel.AddOverriteAsync method, which changes a channels permissions.
